src/cordova/info.js

```diff
@@ -97,16 +97,16 @@ function getPlatformInfo (platform) {
     switch (platform) {
     case 'ios':
         return failSafeSpawn('xcodebuild', ['-version'])
-            .then(out => 'iOS platform:\n' + indent(out));
+            .then(data => 'iOS platform:\n' + indent(data.stdout));
     case 'android':
         return failSafeSpawn('android', ['list', 'target'])
-            .then(out => 'Android platform:\n' + indent(out));
+            .then(data => 'Android platform:\n' + indent(data.stdout));
     }
 }
 
 function failSafeSpawn (command, args) {
-    return superspawn.spawn(command, args)
-        .catch(err => `ERROR: ${err.message}`);
+    return execa(command, args)
+        .catch(error => `ERROR: ${error.stderr}`);
```

Comment:
I another case You replaced `err` with `err.message`. Now here `err.message` with `err.stderr`. How do the errors compare? Can't we just always print the error?